db.go

```diff
@@ -236,7 +240,7 @@ func (s *ColumnStore) DatabasesDir() string {
 }
 
 // ReplayWALs replays the write-ahead log for each database.
```

Comment:
nit: I should've probably renamed this to something like `recoverDBsFromStorage` since it's not just the WAL anymore.